### ISSUE TITLE
Suport in Windows

### ISSUE THREAD

**Issue by felipenmoura** [NascHQ/termtools#2]
This was built using a Mac and we had access to some linux machines.

Would a good soul help us out here, making it work in some Windows terminal simulators?

**Comment by renanbastos93** [comment]
Hello, I did test with bash of windows (there that generate when install the git).
So it show some errors:

![image](https://user-images.githubusercontent.com/8202898/36459715-a3ee3ed2-1693-11e8-8cb5-305236d68ad9.png)

I thinking that need of a *.bat

see ya

**Comment by felipenmoura** [comment]
Yeah, I haven't test it on windows so far. I expected some weird errors.
In this case, it looks like you don't have the global command `termtools` installed (even though it's happening after the installation).
I will have to give it a try in some windows machine and yes, it will probably need quite some work to be done.
All the reports and feedback is welcome :)

**Comment by renanbastos93** [comment]
Oh, okay.

I remake a new test, using the bash of developer mode in window 10. It use one OS Ubuntu for bellow.
So, worked will.

see ya

**Comment by sky0matic** [comment]
Actually @felipenmoura the real issue is not that `termtools` is not global .. it's more that we cannot execute `.sh` files on Windows.

Probably trying to make a copy of the `.sh` into `.bat`'s could allow us to install it in Git Bash for example.

If not then at least we would have a more precise error

**Comment by LuisKrotz** [comment]
In windows is possible to Use the Windows 10 to install an Ubuntu or other Linux bash.
https://www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/

Maybe it helps.
